clusters/build-clusters/build05/ofcir/ofcir.yaml

```diff
@@ -0,0 +1,569 @@
+  - ofcir.openshift
+  resources:
+  - cipools/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - ofcir.openshift
+  resources:
+  - ciresources
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ofcir.openshift
+  resources:
+  - ciresources/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - ofcir.openshift
+  resources:
+  - ciresources/status
+  verbs:
+  - get
+  - patch
+  - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ofcir-metrics-reader
+rules:
+- nonResourceURLs:
+  - /metrics
+  verbs:
+  - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ofcir-proxy-role
+rules:
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - create
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - create
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ofcir-leader-election-rolebinding
+  namespace: ofcir-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ofcir-leader-election-role
+subjects:
+- kind: ServiceAccount
+  name: ofcir-controller-manager
+  namespace: ofcir-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ofcir-manager-rolebinding
+  namespace: ofcir-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ofcir-manager-role
+subjects:
+- kind: ServiceAccount
+  name: ofcir-controller-manager
+  namespace: ofcir-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ofcir-proxy-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: ofcir-proxy-role
+subjects:
+- kind: ServiceAccount
+  name: ofcir-controller-manager
+  namespace: ofcir-system
+---
+apiVersion: v1
+data:
+  controller_manager_config.yaml: |
+    apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+    kind: ControllerManagerConfig
+    health:
+      healthProbeBindAddress: :8081
+    metrics:
+      bindAddress: 127.0.0.1:8080
+    webhook:
+      port: 9443
+    leaderElection:
+      leaderElect: true
+      resourceName: 7c25506c.openshift
+kind: ConfigMap
+metadata:
+  name: ofcir-manager-config
+  namespace: ofcir-system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: ofcir-controller-manager-metrics-service
+  namespace: ofcir-system
+spec:
+  ports:
+  - name: https
+    port: 8443
+    protocol: TCP
+    targetPort: https
+  selector:
+    control-plane: controller-manager
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: ofcir-service
+  namespace: ofcir-system
+spec:
+  ports:
+  - name: http
+    port: 8087
+    protocol: TCP
+  selector:
+    control-plane: controller-manager
+  type: NodePort
```

Comment:
I think this should work (ie visible by normal Prow jobs running on build05 cluster) also by setting a `ClusterIP` just to limit internally the visibility

Reply:
done